docs/source/contributing.rst

```diff
@@ -0,0 +1,3 @@
+.. highlight:: sh
```

Comment:
Let's do this...let's leave it as you have it since it's a bit of chicken-egg (which came first). There are folks that will look at CONTRIBUTING.rst first.

I'm curious, as a frequent train rider myself, about your use case. I typically clone whichever repos that I'm interested in before the 6 hour train ride. This way I have although the source files and doc sources. Do you have a workflow that you like better?